Topics (#11)
gohiring/ptt: README.md

  end
end

# In this case `Handler#call` will be called each time when a new message with a `foo` routing key
# gets delivered
PTT.configure do |pneumatic_tube|
  pneumatic_tube.register_handler('foo', Handler.new)
end
```

> There is only one requirement to handlers, they must response to `#call` method.

Reviewer: ```suggestion
There is only one requirement to handlers, they must respond to `#call`.
```

There are a few types of registers:

 - `register_handler` receives a JSON string which get decoded and passed as a plain Ruby object
 to the handler. It can only consume [direct messages](https://www.cloudamqp.com/blog/2015-09-03-part4-rabbitmq-for-beginners-exchanges-routing-keys-bindings.html#direct-exchange).

>  - `register_row_handler` receives a message as it was given and passes to the handler without any

Reviewer: ```suggestion
 - `register_raw_handler` receives a message as it was given and passes to the handler without any
```

4. Push to the branch (`git push origin my-new-feature`)
5. Create a new Pull Request

> (c) 2020 GOhiring team

Reviewer: ```suggestion
(c) 2020 GoHiring GmbH
```

We're supposed to use the registered company name for copyrights 🙂
